tests/commands/record/test_case_event.py

```diff
@@ -0,0 +1,35 @@
+
+import unittest
+from io import StringIO
+from unittest import mock
+
+from launchable.commands.record.case_event import CaseEvent
+
+UNKNOWN_TIMEZONE_WARNING = "UnknownTimezoneWarning"
+
+
+class TestCaseEventTimestamp(unittest.TestCase):
+    def test_timestamp_with_known_abbreviation(self):
+        # Use a known abbreviation from COMMON_TIMEZONES
+        ts = "2024-06-23T12:34:56.789 PDT"  # PDT is in COMMON_TIMEZONES
+        with mock.patch('sys.stderr', new=StringIO()) as stderr:
+            result = CaseEvent.create(
+                test_path=[], duration_secs=1.0, status=CaseEvent.TEST_PASSED, timestamp=ts
+            )
+            # Should parse to ISO format with correct offset
+            self.assertTrue(result["createdAt"].startswith("2024-06-23T12:34:56.789"))
```

Comment:
Just a question: Why did you use `assertTrue` and `assertIn` instead of `assertEqual(result["createdAt"], "2024-06-23T12:34:56.789 -7:00")`?

Reply:
I just feel like it's clearer to test them separately. The first line, `assertTrue` guarantees whether the time isn't changed. The second line, `assertIn` guarantees that the timezone is parsed correctly.

But, I don't have a strong opinion, so I'm gonna change that so that it uses `assertEqual(result["createdAt"], "2024-06-23T12:34:56.789 -7:00")`

Reply:
[bc99065](https://github.com/launchableinc/cli/pull/1043/commits/bc99065c8d722a6e544e49c04a575ca3234e1cd3)